src/store/modules/aeternity.js

```diff
@@ -261,32 +261,46 @@ export default {
       return decodedResult;
     },
     async tokenSaleMethod(
-      { dispatch },
+      { dispatch, state: { useSdkWallet } },
       {
         contractAddress,
         method,
         args = [],
         options = {},
+        withDeepLink = false,
```

Comment:
is this param for testing purposes? I don't see reasons to use deeplinks if a sdk-based wallet is available